Traceback (most recent call last):
  File "/home/ubuntu/miniconda3/envs/prepa/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/home/ubuntu/miniconda3/envs/prepa/lib/python3.9/site-packages/nbclient/client.py", line 1112, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/ubuntu/miniconda3/envs/prepa/lib/python3.9/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/ubuntu/miniconda3/envs/prepa/lib/python3.9/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/home/ubuntu/miniconda3/envs/prepa/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/home/ubuntu/miniconda3/envs/prepa/lib/python3.9/site-packages/nbclient/client.py", line 553, in async_execute
    await self.async_execute_cell(
  File "/home/ubuntu/miniconda3/envs/prepa/lib/python3.9/site-packages/nbclient/client.py", line 857, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/home/ubuntu/miniconda3/envs/prepa/lib/python3.9/site-packages/nbclient/client.py", line 760, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# On doit normaliser le signal pour que les valeurs soit comprises entre -1 et 1
# En pratique, on place les valeurs entre -0.5 et 0.5 pour Ã©viter tout risque de saturation
reponse2 = []
reponse2.append(s_gauche / np.max(s_gauche) * 0.5)
reponse2.append(s_droite / np.max(s_droite) * 0.5)
reponse2 = np.array(reponse2)  # La fonction write impose l'utilisation d'un array


filesortie = 'signal_filtree.wav'  # Nom du fichier, il sera dans le mÃªme rÃ©pertoire que votre notebook
write(filesortie, samplerate, reponse2)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
[0;32m<ipython-input-4-fcec01014772>[0m in [0;36m<module>[0;34m[0m
[1;32m      2[0m [0;31m# En pratique, on place les valeurs entre -0.5 et 0.5 pour Ã©viter tout risque de saturation[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[1;32m      3[0m [0mreponse2[0m [0;34m=[0m [0;34m[[0m[0;34m][0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 4[0;31m [0mreponse2[0m[0;34m.[0m[0mappend[0m[0;34m([0m[0ms_gauche[0m [0;34m/[0m [0mnp[0m[0;34m.[0m[0mmax[0m[0;34m([0m[0ms_gauche[0m[0;34m)[0m [0;34m*[0m [0;36m0.5[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      5[0m [0mreponse2[0m[0;34m.[0m[0mappend[0m[0;34m([0m[0ms_droite[0m [0;34m/[0m [0mnp[0m[0;34m.[0m[0mmax[0m[0;34m([0m[0ms_droite[0m[0;34m)[0m [0;34m*[0m [0;36m0.5[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m      6[0m [0mreponse2[0m [0;34m=[0m [0mnp[0m[0;34m.[0m[0marray[0m[0;34m([0m[0mreponse2[0m[0;34m)[0m  [0;31m# La fonction write impose l'utilisation d'un array[0m[0;34m[0m[0;34m[0m[0m

[0;31mNameError[0m: name 's_gauche' is not defined
NameError: name 's_gauche' is not defined

